students/tony7kh/hw-4/answer.js

```diff
@@ -0,0 +1,24 @@
+function searchVowels(initialString) {
+    let vowels = 'aeiouyAEIOUY';
+    let vowelsCounter = 0;
+    let normalizeString = initialString.toLowerCase() ; 
+    for(let i = 0; i < normalizeString.length ; i++) {
+        if (vowels.indexOf(normalizeString[i]) !== -1) {
+        vowelsCounter += 1;
+        }
+    
+    }
+    return vowelsCounter;
+}
+
+function inputValidation(input) {
+    const validNumber = /(?:\+|\d)[\d\-\(\) ]{9,}\d/ ;
+    const validMail = /^([a-z0-9_-]+\.)*[a-z0-9_-]+@[a-z0-9_-]+(\.[a-z0-9_-]+)*\.[a-z]{2,6}$/;
```

Comment:
There is missed validation for the second part of the mail. And it should be impossible to add digits after `@` symbol.